ansible-ipi-install/roles/installer/tasks/04_create_metal3.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Get COMMIT_ID
+  shell: |
+    {{ extract_dir }}/openshift-baremetal-install version | grep '^built from commit' | awk '{print $4}' 
+  register: commit_id
+
+- name: Get RHCOS_URI
+  shell: |
```

Comment:
Nit: I'd say to use ansible's filters and such to avoid doing so much shell commands (such as https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#json-query-filter )

Reply:
Maybe you can assist me with this once we PR? I honestly haven't use ansible filters that much